examples/model_with_vllm_test.py

```diff
@@ -1,16 +1,33 @@
 # SPDX-License-Identifier: Apache-2.0
 
+import os
+import random
 import time
 
+import numpy as np
+import torch
 from vllm import LLM, SamplingParams
 
 import flag_gems
 
+os.environ["VLLM_ENABLE_MOE_ALIGN_BLOCK_SIZE_TRITON"] = "1"
```

Comment:
Why is this used?

Reply:
We currently lack a native MOE implementation, so this setting lets us pick a Triton implementation for now.
But I agree, it doesn’t belong in `model_with_vllm_test.py`

Reply:
Is "pick a Triton implementation" necesasary for testing? Or is it optional to use triton implementation?